lib/siwapp_web/resolvers/invoice.ex

```diff
@@ -89,6 +89,11 @@ defmodule SiwappWeb.Resolvers.Invoice do
     end
   end
 
+  @spec get_series_code(Invoices.Invoice.t(), map(), Absinthe.Resolution.t()) :: {:ok, String.t()}
+  def get_series_code(invoice, _args, _resolution) do
+    {:ok, Repo.preload(invoice, :series).series.code}
```

Comment:
The series is also preloaded for non-draft invoices by `set_reference/1`, so in most cases, this resolver won't make any database round-trips